apps/wallet-dashboard/components/dialogs/staking/views/EnterTimelockedAmountView.tsx

```diff
@@ -53,18 +50,18 @@ export function EnterTimelockedAmountView({
     const [possibleAmount, setPossibleAmount] = useState<bigint | null>(null);
     const [isSearchingProtocolMaxAmount, setSearchingProtocolMaxAmount] = useState(false);
 
-    const { data: currentEpochMs } = useGetCurrentEpochStartTimestamp();
+    const { data: clockTimestampMs } = useGetClockTimestamp();
     const { data: timelockedObjects } = useGetAllOwnedObjects(senderAddress, {
         StructType: TIMELOCK_IOTA_TYPE,
     });
     const groupedTimelockObjects = useMemo(() => {
-        if (!timelockedObjects || !currentEpochMs || possibleAmount === null) return [];
+        if (!timelockedObjects || !clockTimestampMs || possibleAmount === null) return [];
```

Comment:
We should probably remove these `!clockTimestampMs`, because `0` is a falsy value.